src/hotspot/cpu/s390/sharedRuntime_s390.cpp

```diff
@@ -2755,7 +2754,8 @@ void SharedRuntime::generate_uncommon_trap_blob() {
   // set the "unpack" frame as last_Java_frame.
   // `Deoptimization::uncommon_trap' expects it and considers its
   // sender frame as the deoptee frame.
-  __ get_PC(Z_R1_scratch);
+  int offs = __ offset();
```

Comment:
`generate_uncommon_trap_blob()` has no oop_maps on s390. Most other platforms have them. I think this part of the change doesn't make sense without oop_maps.

Reply:
Thanks for your comments, I have Addressed this comment in another commit.